client/src/views/page-components/account-page/AccountHistoryModal/AccountHistoryModal.ts

```diff
@@ -266,7 +264,7 @@ export default class AccountHistoryModal {
   onClickCategory(e: MouseEvent) {
     const { target } = e;
     if (!(target instanceof HTMLElement)) return;
-    if (target.id === 'category-item') {
+    if (target.className === 'category-item') {
       const allCategoryElement = document.querySelectorAll('#category-outline');
```

Comment:
id 선택자를 쓰셨네요 id 선택자는 유일하게 존재해야 하므로 querySelectorAll method는 적합하지 않아 보입니다.
만약 , querySelector를 쓸거면 만들어둔 util 함수를 사용하는 것이 좋아 보입니다.

```suggestion
      const allCategoryElement = &('#category-outline');
```

Reply:
왜 저렇게 했을까요.. 흠 수정할게욥